Cabal/ChangeLog.md

```diff
@@ -0,0 +1,784 @@
+    #5050).
+  * Support for GHC's numeric -g debug levels (#4673).
+  * Compilation with section splitting is now supported via the
+    '--enable-split-sections' flag (#4819)
+  * Fields with mandatory commas (e.g. build-depends) may now have a
+    leading or a trailing comma (either one, not both) (#4953)
+  * Added `virtual-modules` field, to allow modules that are not built
+    but registered (#4875).
+  * Use better defaulting for `build-type`; rename `PackageDescription`'s
+    `buildType` field to `buildTypeRaw` and introduce new `buildType`
+    function (#4958)
+  * `D.T.PackageDescription.allBuildInfo` now returns all build infos, not
+    only for buildable components (#5087).
+  * Removed `UnknownBuildType` constructor from `BuildType` (#5003).
+  * Added `HexFloatLiterals` to `KnownExtension`.
+  * Cabal will no longer try to build an empty set of `inputModules`
+    (#4890).
+  * `copyComponent` and `installIncludeFiles` will now look for
+    include headers in the build directory ('dist/build/...' by
+    default) as well (#4866).
+  * Added 'cxx-options' and 'cxx-sources' buildinfo fields for
+    separate compilation of C++ source files (#3700).
+  * Removed unused '--allow-newer'/'--allow-older' support from
+    'Setup configure' (#4527).
+  * Changed `FlagAssignment` to be an opaque `newtype` (#4849).
+  * Changed `rawSystemStdInOut` to use proper type to represent
+    binary and textual data; new 'Distribution.Utils.IOData' module;
+    removed obsolete 'startsWithBOM', 'fileHasBOM', 'fromUTF8',
+    and 'toUTF8' functions; add new `toUTF8BS`/`toUTF8LBS`
+    encoding functions. (#4666)
+  * Added a 'cabal check' warning when the `.cabal` file name does
+    not match package name (#4592).
+  * The 'ar' program now receives its arguments via a response file
+    ('@file').  Old behaviour can be restored with
+    '--disable-response-files' argument to 'configure' or
+    'install' (#4596).
+  * Added '.Lens' modules, with optics for package description data
+    types (#4701).
+  * Support for building with Win32 version 2.6 (#4835).
+  * Change `compilerExtensions` and `ghcOptExtensionMap` to contain
+    `Maybe Flag`s, since a supported extention can lack a flag (#4443).
+  * Pretty-printing of .cabal files is slightly different due to
+    parser changes. For an example, see
+    https://mail.haskell.org/pipermail/cabal-devel/2017-December/010414.html.
+  * `--hyperlink-source` now uses Haddock's hyperlinker backend when
+    Haddock is new enough, falling back to HsColour otherwise.
+  * `D.S.defaultHookedPackageDesc` has been deprecated in favour of
+    `D.S.findHookedPackageDesc` (#4874).
+  * `D.S.getHookedBuildInfo` now takes an additional parameter
+    specifying the build directory path (#4874).
+  * Emit warning when encountering unknown GHC versions (#415).
+  * TODO
+
+### 2.0.1.1 Mikhail Glushenkov <mikhail.glushenkov@gmail.com> December 2017
```

Comment:
GitHub's renderer removes angle brackets here for some reason, would be nice to fix this.

Reply:
That's a feature of markdown called "autolink"; something like `<url>` or `<email-addr>` is turned into a link w/ the address as label

Reply:
Maybe then it's worth doing more "normal" Markdown links `[Name](email)`? Slightly less readable as text, more readable when rendered?